server/utils/nunjucksSetup.ts

```diff
@@ -166,24 +176,20 @@ export default function nunjucksSetup(app: express.Application): nunjucks.Enviro
           },
           clearLink: {
             text: 'Clear filters',
+            href: '/court-register/paged',
           },
           categories: [
             {
               heading: {
-                text: 'Court Types',
+                text: 'Active?',
```

Comment:
Maybe this should be "Open or Closed". I think we use Open rather than Active in UI